app/ante/ante.go

```diff
@@ -91,17 +94,27 @@ func NewAnteHandler(options HandlerOptions) (sdk.AnteHandler, error) {
 	}, nil
 }
 
-func newCosmosAnteHandler(options HandlerOptions) sdk.AnteHandler {
+func newCosmosAnteHandler(options HandlerOptions, isEIP712 bool) sdk.AnteHandler {
```

Comment:
is it possible to add `IsEIP712` as a field on the existing `HandlerOptions` struct?

Reply:
`HandlerOptions` is shared options for all ante handlers that gets passed during app setup so we can't put this option there. I can take a look at other refactors to clean up the params here.